Index: event/listener.php
===================================================================
--- event/listener.php	(revision 680)
+++ event/listener.php	(working copy)
@@ -483,7 +483,7 @@
 					}
 				}
 			} else
-				$row['next_beg_time'] = 1234567890;
+				$row['next_beg_time'] = 1234567890 + $row['gym_jour'];
 
 			// Horaires
 			$row['gym_heure'] = substr('00'.$row['gym_heure'], -2);
